1.3/Patches/Patches.xml

```diff
@@ -760,37 +787,84 @@
 							<li>vision_adj->colorful</li>
 							<li>vision_adj->clear</li>
 							<li>vision_adj->sudden</li>
-
+							<li>vision_adj->vivid</li>
+							<li>vision_adj->curious</li>
+							<li>vision_adj->hazy</li>
+							<li>vision_adj->flash of a</li>
 						
 				<!-- ===CHITCHAT WILDCARDS: dictionary=== -->
+					<!-- ===animal — cute=== -->
+						<li>animal_cute->bunny</li>
+						<li>animal_cute->chinchilla</li>
+						<li>animal_cute->honeybee</li>
+						<li>animal_cute->lamb</li>
+						<li>animal_cute->duckling</li>
 					<!-- ===animal — large=== -->
 						<li>animal_large->a horse</li>
 						<li>animal_large->a muffalo</li>
 						<li>animal_large->an elephant</li>
+						<li>animal_large(INITIATOR_intellectual_level&lt;7)->a rhino</li>
+						<li>animal_large(INITIATOR_intellectual_level>=7)->a rhinoceros</li>
+						<li>animal_large->a whale</li>
+						<li>animal_large->a grizzly bear</li>
+						<li>animal_large->a megasloth</li>
 					<!-- ===annoying=== -->
 						<li>annoying->annoying</li>
 						<li>annoying->bad</li>
 						<li>annoying->irritating</li>
 						<li>annoying->infuriating</li>
 						<li>annoying->maddening</li>
 						<li>annoying->bothersome</li>
+						<li>annoying->frustrating</li>
+						<li>annoying(INITIATOR_intellectual_level&lt;5)->[intensifier]annoying</li>
+						<li>annoying(INITIATOR_intellectual_level&lt;5)->[intensifier]bad</li>
 						<li>annoying(INITIATOR_intellectual_level>=5)->exasperating</li>
+						<li>annoying(INITIATOR_intellectual_level>=5)->ire-raising</li>
+						<li>annoying(INITIATOR_intellectual_level>=5)->vexing</li>
+					<!-- ===beautiful (person)=== -->
+						<li>beautiful_person->adorable</li>
+						<li>beautiful_person->alluring</li>
+						<li>beautiful_person->attractive</li>
+						<li>beautiful_person->beautiful</li>
+						<li>beautiful_person->cute</li>
+						<li>beautiful_person->good-looking</li>
+						<li>beautiful_person->gorgeous</li>
+						<li>beautiful_person->lovely</li>
+						<li>beautiful_person->pretty</li>
+						<li>beautiful_person->statuesque</li>
```

Comment:
Wouldn't it be better to use this only for high intellectuals?

Reply:
Statuesque was available for high intellectual pawns, I forgot to remove the normal pawn usage. Thanks! Removing!